.circleci/real_config.yml

```diff
@@ -2158,6 +2158,8 @@ jobs:
       - setup-python-venv:
           executor: <<pipeline.parameters.machine-image>>
       - setup-go-intg-deps
+      - run: curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && sudo install minikube-linux-amd64 /usr/local/bin/minikube
```

Comment:
Maybe not a thing for this PR, but can we move this (and maybe the start too) out to a command so we don't have a proliferation of these huge command lines everywhere minikube is used? Should I open that as an issue somewhere?

Reply:
sounds like a job for infra

Reply:
yeah can do. but fyi this is the only such line, I cherry picked from carolina's PR you reviewed. it'll go away once she lands and i rebase. mb, should've waited to mark it ready for review. would've saved you a bit of time.

Reply:
Ah, ok. Cool.

Reply:
fyi, I merged the PR, so you can rebase against main to pick it up!